app/src/main-process/log.ts

```diff
@@ -1,6 +1,6 @@
-import * as Fs from 'fs-extra'
 import * as Path from 'path'
 import * as winston from 'winston'
+import * as fileSystem from '../lib/file-system'
```

Comment:
Can you make this `import { mkdirIfNeeded } from '../../lib/file-system'` to be more explicit here?